xcode/Subconscious/Shared/Models/MemoDetailResponse.swift

```diff
@@ -10,7 +10,6 @@ import Foundation
 /// Response when reading info for a view-only memo detail from data source.
 struct MemoDetailResponse: Hashable {
     var entry: MemoEntry
-    var backlinks: [EntryStub] = []
```

Comment:
Since this struct only carries an entry now, should we just respond with a MemoEntry?